### ISSUE TITLE
[Adaptive cards - Action.ShowCard.Style.json]: Voice over focus is not landing on text 'This card has no style' which appears after activating 'No Style' button.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8417]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Activate Voice over
3. Home screen will appear.
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Action.ShowCard.Style.json' under 'Elements' list item and activate it.
7. Card will be appear.
8. Navigate to 'No style' button and activate it
9. Observe the issue.
 
**Actual Result:**
After activating the 'No Style' button, VO is not conveying the status message as ''This card has no style''.

**Expected Result:**
Voice over focus should announce the status message to the user while activating the 'No Style' button and it should announce as ''This card has no style''.

**Note:**
The issue is also repro'ing for text appearing after activating 'default style' and 'Emphasis' style.

**User Experience:**
Voice over users will be impacted if Voice over will not convey the status message to them as they will not aware whether the control has been updated or not?

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/status-messages

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/229733689-67b63310-92ff-4037-8195-c4b5e68ca6e3.mov

### Card JSON

```JSON
NA
```
